### ISSUE TITLE
Manifest tag doesn't accept variable argument

### ISSUE THREAD

**Issue by knokit** [shonin/django-manifest-loader#17]
Trying to use a variable filename:

``` django
{% with page|add:".css" as file %}
    <script src="{% manifest file %}" />
{% endwith %}
```

Results in TemplateSyntaxError: `'manifest' tag's argument should be in quotes`.

**Comment by shonin** [comment]
Hi @knokit thank's for the report. Unfortunately, the fix requires more of a refactor than I anticipated, as the `context` is not available where the calculations were previously being done. I'm hoping to have something pushed out in the next couple of days.

**Comment by shonin** [comment]
@knokit this should be fixed for `0.1.0`. Thanks again for the report!
